content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,846 @@
+
+Let's add the motion vector shader and then edit its vertex shader:
+
+1. Open `VegetationBending.materialtype`. At the bottom of the `shaders` list, add the motion vector pass:
+   
+   ```json
+   {
+      "file": "./MeshMotionVectorVegetationBending.shader",
+      "tag": "MeshMotionVector"
+   }
+   ```
+
+1. Open `MeshMotionVectorVegetationBending.azsl`.
+
+1. Add the call to `ProcessBending`, just as you did in the previous steps with the other shaders. Under the declaration for `float4 prevWorldPosition`, but above `OUT.m_worldPos`, add:
+
+   ```glsl
+   float currentTime = SceneSrg::m_time;
+   worldPosition = ProcessBending(currentTime, IN.m_position, IN.m_normal, IN.m_optional_color, worldPosition, objectToWorld);
+   ```
+
+1. For motion vectors to work you need to find the previous world position. You will want to perform bending on the vertex at the `IN.m_position` but at the previous frame time, with the previous world position. You can use `SceneSrg::m_prevTime` to get the previous frame time. Call your `ProcessBending` function again, but with the appropriate time and world position:
+
+   ```glsl
+   float currentTime = SceneSrg::m_time;
+   worldPosition = ProcessBending(currentTime, IN.m_position, IN.m_normal, IN.m_optional_color, worldPosition, objectToWorld);
+   float prevTime = SceneSrg::m_prevTime;
+   prevWorldPosition = ProcessBending(prevTime, IN.m_position, IN.m_normal, IN.m_optional_color, prevWorldPosition, prevObjectToWorld);
+   ```
+
+1. Take a look at the pixel shader to see how the motion vector is calculated! There is no need to edit the pixel shader.
+
+Amazing, you have added everything you need to add for motion vectors! However, if you open the **Editor** and just view the tree, you'll see that there is no difference. You can, however, observe that the motion vector pass works by using the **pass tree visualizer**.
+1. Open the Editor and press **Ctrl-G** to enter gameplay mode.
+
+1. Press the **Home** key on your keyboard. This brings up the toolbar at the top.
+
+1. Select **Atom Tools** > **Pass Viewer**.
+
+1. In the pop-up **PassTree View**, enable **Preview Attachment** and **Show Pass Attachments**.
+
+1. In the **PassTree**, find *MotionVectorPass* > *MeshMotionVectorPass* and select the line with `CameraMotion`. 
+
+1. Ensure you are viewing your tree. You probably won't see much except black on the bottom left preview, because the motion vectors are very small since the tree only moves minimally. 
+
+   However, if you move the camera around or translate the tree quickly, you may see some motion vectors pop up. You can also open `MeshMotionVectorVegetationBending.azsl` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
```

Comment:
```suggestion
1. Ensure you are viewing your tree. 
   
   {{< note >}}
   You may only see black on the bottom left preview, because the motion vectors are small, meaning the tree moves minimally. 

   To better see the motion vectors, you can move the camera around or translate the tree quickly. You can also open `MeshMotionVectorVegetationBending.azsl` and scale `OUT.m_motion` in the pixel shader to ensure that the motion vectors' directions are working properly.
   {{< /note >}}
```